aidants_connect_web/tests/test_views.py

```diff
@@ -162,6 +162,41 @@ def test_post_to_recap_without_sub_creates_error(self):
         messages = list(get_messages(response.wsgi_request))
         self.assertEqual(len(messages), 1)
 
+    def test_post_to_recap_checks_existing_sub_and_updates_info(self):
```

Comment:
I don't think we *want* to update the information.
If any information changes in the user info, the FC sub would change anyway (I checked)
I think the behavior should be to create or do nothing if the sub exist.
I think it might be overkill to throw an error if the sub exist and the information provided is different from the one we have in store as the risk of two identical subs for two different people seems low.